### ISSUE TITLE
docs: Overview

### ISSUE THREAD

**Issue by withinboredom** [jcmoraisjr/haproxy-ingress#965]
On the homepage it says:

```
The first and most complete ingress controller implementation for HAProxy.
```

Yet the very first page of documentation has two `TODO` which doesn't lend much confidence to the project.

**Comment by jcmoraisjr** [comment]
Yep, I'm evolving and trying to do the best I can only in the most important parts of the documentation, the ones that users need to use to make their jobs done. But yeah in fact those TODOs doesn't seem that professional. Maybe someone could help us, write something nice and fix that for us =)
